File: login_and_signup_web-master/lib/Model/ListVendors.dart
class ListVendors {
  int vendorID;
  String vendorName;
  String vendRef;
  bool isActive;
  int lastUpdatedBy;
  String lastUpdatedDT;
  ListVendors(
      {this.vendorID,
      this.vendorName,
      this.vendRef,
      this.isActive,
      this.lastUpdatedBy,
      this.lastUpdatedDT});

  factory ListVendors.fromJson(Map<String, dynamic> json) {
    return ListVendors(
        vendorID: json['VendorId'],
        vendorName: json['VendorName'],
        vendRef: json['VendRef'],
        isActive: json['IsActive'],
        lastUpdatedBy: json['LastUpdatedBy'],
        lastUpdatedDT: json['LastUpdatedDT']);
  }
}

File: AniDAAPI/AniDAAPI/Models/ListVendors.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AniDAAPI.Models
{
    public class ListVendors
    {
		public int VendorId { get; set; }
		public string VendorName { get; set; }
		public string VendRef { get; set; }
		public bool? IsActive { get; set; }
		public int? VendTypeId { get; set; }
		public int? LastUpdatedBy { get; set; }
		public DateTime? LastUpdatedDT { get; set; }
}
}
